### ISSUE TITLE
Close Item info overlay

### ISSUE THREAD

**Issue by stefaz** [Lailloken/Lailloken-UI#131]
I read in one of the other issues that the item info overlay should close automatically if I activate the PoE window again. for me this doesn't work, I always have to press ESC to close the overlay even if I'm already back in the game (running around / interacting with NPCs, stash). Is this working as intended or am I missing something?

**Comment by Lailloken** [comment]
Right now, it only automatically closes if you release shift while using it in conjunction with wisdom scrolls. I have to check again if there was a reason for not closing automatically like you describe (there might be one, or it's just an oversight on my part). If there isn't, I can change it for the next release.

**Comment by Lailloken** [comment]
Okay, I checked it again and now I can remember why it behaves like that. If we want it to close automatically when activating the game-client, the tooltip needs to be the active window and the game-client needs to be in the background -- if the client is activated, the tooltip closes, sounds good. But if I do that, you won't be able to open another tooltip while the first is active because the client doesn't let you copy item information while it's in the background. You would have to press the omni-key twice every time to make it work correctly.

I tried some workarounds but those didn't work either, resulting in the same double-press behavior. The next release introduces a feature that relies on recognizing if the inventory is open or not, so I could link the inventory-state to the tooltip and close it whenever the inventory closes.
